README.md

```diff
@@ -1 +1,134 @@
-# Afghnistan-Developers-Conference
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [Afghanistan Developers Confrence] <a name="about-project"></a>
+
+> You can find my recent works here. This is also a direct way to contact with me
+
+**[Afghanistan Developers Confrence]** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+HTML 
+CSS 
+JAVASCRIPT
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">JAVASCRIPT</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Github Pages</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Home_Page]**
+- **[About_Page]**
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://kazim110.github.io)
+
+## 💻 Loom Video for Introduction<a name="loom-video"></a>
+
+- [Loom Video Link](https://www.loom.com/share/b374ceff82ef4ba18a0605b8a6dfe9c3)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+ First Clone it 
+ run npm install
+ run npm start
+
+
+### Prerequisites
+
+In order to run this project you need:
+npm
+
+
+### Setup
+
+Clone this repository to your desired folder:
+git clone https://github.com/kazim110/Afghnistan-Developers-Conference.git
+
+### Install
+
+Install this project with:
+npm install
+
+
+### Usage
+
+To run the project, execute the following command:
+npm start
+
+
+
+
+## 👥 Authors <a name="authors">Mohammad Kazim</a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/kazim110)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kazim-mohammadi-89aa4a1aa/)
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Additional_Animation]**
+- [ ] **[Powerfull_UI]**
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐ if you like this project!
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgements">Cindy Shin</a>
+
+- Design has credited by Candy Shin
```

Comment:
- [ ] Good job on adding this required acknowledgment, however, her name is Cindy Shin not Candy 😅 and it would be better if you added the link to the design with this as shown by the previous reviewer 👍